FinSearchUnified/Tests/Bundle/SearchBundleFindologic/NavigationQueryBuilderTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace FinSearchUnified\Tests\Bundle\SearchBundleFindologic;
+
+use Exception;
+use FinSearchUnified\Bundle\SearchBundleFindologic\NavigationQueryBuilder;
+use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder;
+use PHPUnit\Framework\Assert;
+use Shopware\Bundle\PluginInstallerBundle\Service\InstallerService;
+use Shopware\Components\HttpClient\GuzzleHttpClient;
+use Shopware\Components\HttpClient\Response;
+use Shopware\Components\Test\Plugin\TestCase;
+use Shopware_Components_Config;
+
+class NavigationQueryBuilderTest extends TestCase
+{
+    /**
+     * @var Shopware_Components_Config
+     */
+    private $config;
+
+    /**
+     * @var InstallerService
+     */
+    private $installerService;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->config = Shopware()->Container()->get('config');
+        $this->installerService = Shopware()->Container()->get('shopware_plugininstaller.plugin_manager');
+        Shopware()->Session()->offsetSet('isSearchPage', false);
```

Comment:
Implement a tearDown method (should be placed right after setUp) which unsets `isSearchPage`.